### ISSUE TITLE
Strange Consul leader election error

### ISSUE THREAD

**Issue by siddharthist** [CiscoCloud/mantl#1190]
While deploying sample.yml from master, I got the following error:
```
TASK: [consul | wait for leader] ********************************************** 
failed: [lb-mi-worker-003] => {"changed": true, "cmd": ["/usr/local/bin/consul-wait-for-leader.sh"], "delta": "0:00:35.069108", "end": "2016-03-01 19:12:26.991073", "rc": 1, "start": "2016-03-01 19:11:51.921965", "warnings": []}
stderr: Unexpected response code: 500 (No known Consul servers)
Unexpected response code: 500 (No known Consul servers)
Unexpected response code: 500 (No known Consul servers)
Unexpected response code: 500 (No known Consul servers)
Unexpected response code: 500 (No known Consul servers)
Unexpected response code: 500 (No known Consul servers)
stdout: No Consul leader elected in 30 seconds
failed: [lb-mi-worker-002] => {"changed": true, "cmd": ["/usr/local/bin/consul-wait-for-leader.sh"], "delta": "0:00:35.099333", "end": "2016-03-01 19:12:26.830539", "rc": 1, "start": "2016-03-01 19:11:51.731206", "warnings": []}
stderr: Unexpected response code: 500 (No known Consul servers)
Unexpected response code: 500 (No known Consul servers)
Unexpected response code: 500 (No known Consul servers)
Unexpected response code: 500 (No known Consul servers)
Unexpected response code: 500 (No known Consul servers)
Unexpected response code: 500 (No known Consul servers)
stdout: No Consul leader elected in 30 seconds
failed: [lb-mi-control-03] => {"changed": true, "cmd": ["/usr/local/bin/consul-wait-for-leader.sh"], "delta": "0:00:35.116476", "end": "2016-03-01 19:12:27.120996", "rc": 1, "start": "2016-03-01 19:11:52.004520", "warnings": []}
stdout: No Consul leader elected in 30 seconds
failed: [lb-mi-worker-001] => {"changed": true, "cmd": ["/usr/local/bin/consul-wait-for-leader.sh"], "delta": "0:00:35.104052", "end": "2016-03-01 19:12:26.680189", "rc": 1, "start": "2016-03-01 19:11:51.576137", "warnings": []}
stderr: Unexpected response code: 500 (No known Consul servers)
Unexpected response code: 500 (No known Consul servers)
Unexpected response code: 500 (No known Consul servers)
Unexpected response code: 500 (No known Consul servers)
Unexpected response code: 500 (No known Consul servers)
Unexpected response code: 500 (No known Consul servers)
stdout: No Consul leader elected in 30 seconds
failed: [lb-mi-edge-01] => {"changed": true, "cmd": ["/usr/local/bin/consul-wait-for-leader.sh"], "delta": "0:00:35.080404", "end": "2016-03-01 19:12:27.969111", "rc": 1, "start": "2016-03-01 19:11:52.888707", "warnings": []}
stderr: Unexpected response code: 500 (No known Consul servers)
Unexpected response code: 500 (No known Consul servers)
Unexpected response code: 500 (No known Consul servers)
Unexpected response code: 500 (No known Consul servers)
Unexpected response code: 500 (No known Consul servers)
Unexpected response code: 500 (No known Consul servers)
stdout: No Consul leader elected in 30 seconds
```
Seems off. I'll do some more debugging and post here.

**Comment by ChrisAubuchon** [comment]
How many control nodes do you have?

**Comment by siddharthist** [comment]
Three

**Comment by ChrisAubuchon** [comment]
I only see one in the list. Did any fail prior to this?

**Comment by siddharthist** [comment]
Yeah, it looks like they're failing on `enable docker`, now that you mention it.

**Comment by ChrisAubuchon** [comment]
That'll do it.

**Comment by dcheperd** [comment]
I fixed this in #1585
